src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public abstract class Machine {
+
```

Comment:
<img width="1013" alt="Screenshot 2022-01-17 at 08 58 18" src="https://user-images.githubusercontent.com/33026471/149723463-e5b9e9e0-bd77-427f-9356-abf74e298105.png">
Let's remove all redundant empty lines here and in main